debian/collectd-core.collectd.default

```diff
@@ -12,6 +12,10 @@ USE_COLLECTDMON=1
 # default: 30
 MAXWAIT=30
 
+# try a terminal kill -9 if the process refuses to shut down
+# default: 0
+declare -i USE_SIGKILL=1
```

Comment:
For consistency with other settings, please don't use `declare`. I'm fine with a separate commit changing this globally.